app/styles/modules/_settings.scss

```diff
@@ -70,7 +93,10 @@ body.settings #main-content.card {
 
     @include respond-to('small') {
       margin-top: 9px;
-      width: 29%;
+      html[dir=ltr] & {
+    	 width: 29%;
```

Comment:
I wonder if we could get away not special casing ltr, but instead changing `width` to be `min-width`?